src/main/resources/application.properties

```diff
@@ -3,4 +3,6 @@ spring.data.mongodb.auto-index-creation=true
 server.port=${SERVER_PORT:8081}
 spring.kafka.consumer.group-id=astha
 auth.token.public.key=${PUBLIC_KEY}
-spring.kafka.bootstrap-servers=${KAFKA_BOOTSTRAP_SERVERS}
\ No newline at end of file
+spring.kafka.bootstrap-servers=${KAFKA_BOOTSTRAP_SERVERS}
+springdoc.api-docs.path=/api-docs
+springdoc.swagger-ui.path=/swagger-ui-custom.html
```

Comment:
@astha-singh-2445 
nit: SpringDoc configure default values for these. Check if you actually need these values